terraform/dev/main.tf

```diff
@@ -65,13 +65,13 @@ data "cloudfoundry_service" "s3" {
 resource "cloudfoundry_service_instance" "staticfiles" {
   name             = "tdp-staticfiles-dev"
   space            = data.cloudfoundry_space.space.id
-  service_plan     = data.cloudfoundry_service.s3.service_plans["basic"]
+  service_plan     = data.cloudfoundry_service.s3.service_plans["basic-public-sandbox"]
```

Comment:
This is an important change, the `staticfiles` service must be `public` in order for the client side rendering of Django Admin to work properly. Additionally, by using a sandbox plan for dev and staging we can prevent leaving around orphaned AWS buckets in the case these services are deleted. In production the `sandbox` plan will not be used.